src/Stryker.Core/Stryker.Core/Options/Inputs/OpenReporterInput.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using Stryker.Core.Exceptions;
+
+namespace Stryker.Core.Options.Inputs
+{
+    public enum ReportType
+    {
+        None,
+        HTMLReport
+    }
+
+    public class OpenReporterInput : Input<string>
+    {
+        public override string Default => "None";
+
+        protected override string Description => "The dashboard to open automatically";
+
+        protected override IEnumerable<string> AllowedOptions => Enum.GetNames(typeof(ReportType));
+
+        public ReportType Validate()
```

Comment:
```suggestion
        public ReportType Validate(bool enabled)
```

When the OpenReport option is set/enabled, stryker should automatically open the default report type Html. To be able to do this you will need to capture whether or not the flag is passed to stryker. Since you implemented this option in the CLI using a SingleOrNoValue you have the boolean value available. SingleOrNoValue means that a user can either supply `--open-report` (NoValue) OR `--open-report:Html` (SingleValue). In the case of `--open-report` with no value given, I would like the default to be Html. In the future the SingleValue part of this option can be used to open different kinds of reports. ReportType.None should only be used to show that the option was not turned on (IE the flag was not passed to stryker).

Reply:
See https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.CLI/Stryker.CLI/CommandLineConfigHandler.cs#L93 for an example of the SingleOrNoValue in action.